cfgov/jinja2/v1/_includes/organisms/item-introduction.html

```diff
@@ -54,7 +53,7 @@
     {% endif %}
         {% if has_authors %}
             <span class="byline">
-            {%- for author in page_authors -%}
+            {%- for author in page.authors.order_by('name') -%}
                 {{- 'By ' if loop.index == 1 else ' and ' }}
```

Comment:
Nice, okay. At first I was wondering about why it was necessary but I didn't notice there was two separate sorts going on. Refactored it a bit and pushed it up.